Sources/MapboxMaps/Ornaments/Compass/MapboxCompassOrnamentView.swift

```diff
@@ -52,14 +52,16 @@ internal class MapboxCompassOrnamentView: UIButton {
                                               comment: "Accessibility hint")
 
         containerView.translatesAutoresizingMaskIntoConstraints = false
-        if let image = createCompassImage() {
+        self.originalImage = createCompassImage()
+        if let image = self.originalImage {
             updateImage(image: image)
         }
         addSubview(containerView)
         addTarget(self, action: #selector(didTap), for: .touchUpInside)
     }
 
     func updateImage(image: UIImage?) {
+        let image = image ?? self.originalImage
```

Comment:
I think it's not worth the trouble(<sub>~~memory~~</sub>) retaining the original image - it is unlikely to be reset back to the default, and it can always be recreated.
```suggestion
        let image = image ?? createCompassImage()
```

Reply:
I was thinking about the same 👍